packages/core/test/issue-277.test.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var plugin = require("../plugins/scoping.js"),
+
+    processor = require("postcss")([ plugin ]);
+
+function process(src, options) {
+    return processor.process(
+        src,
+        Object.assign(Object.create(null), {
+            from  : "packages/core/test/specimens/a.css",
+            namer : (file, selector) => `a_${selector}`
+        },
+        options || {})
+    );
+}
+
+describe("/plugins", function() {
```

Comment:
Use `/issues /277` please, like the other issue-specific test files do for their names.